examples/get-started/flutter-for/ios_devs/lib/navigation.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    // create an instance of our app
+    // and passes it to the runApp function
+    const App(),
+  );
+}
+
+// #docregion Routes
+// defines the route name as a constant
+// so that it's reusable
+const detailsPageRouteName = '/details';
+
+class App extends StatelessWidget {
+  const App({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoApp(
+      home: const HomePage(),
+      // The routes property defines the available named routes
+      // and the widgets to build when navigating to those routes
+      routes: {
+        detailsPageRouteName: (context) => const DetailsPage(),
+      },
+    );
+  }
+}
+// #enddocregion Routes
+
+// create a class that holds each person's data
+@immutable
+class Person {
+  final String name;
+  final int age;
+  const Person({
+    required this.name,
+    required this.age,
+  });
+}
+
+// then create a list of people
```

Comment:
Consider making more idiomatic - 'Generates a list of 100 [Person]s' (or '100 people').